crates/pathfinder/src/sequencer/reply.rs

```diff
@@ -443,36 +443,40 @@ pub mod state_update {
     /// L2 contract data within state diff.
     #[derive(Copy, Clone, Debug, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
     #[serde(deny_unknown_fields)]
-    pub struct Contract {
+    pub struct DeployedContract {
```

Comment:
I've added this change to decrease confusion and stick to the current convention, which is:
- always indicating the difference between declared and deployed,
- class_hash is the correct term, contract_hash is the deprecated one.